### ISSUE TITLE
Vertex Correspondence between meshes

### ISSUE THREAD

**Issue by bhattaca** [facebookresearch/pytorch3d#354]
## ❓ Questions on how to use PyTorch3D
Hi Team
Is there a way to find vertex Correspondence between two meshes .
potentially, in the chamfer loss function, is it possible to output the vertices matched in the two meshes. 

Thanks !

**Comment by gkioxari** [comment]
We do return the alignment of two pointclouds when we compute chamfer distance. Even though we don't expose it, it's in 

https://github.com/facebookresearch/pytorch3d/blob/f2eb34dc7a133b319d01d0fe0e208f385498f009/pytorch3d/loss/chamfer.py#L148

https://github.com/facebookresearch/pytorch3d/blob/f2eb34dc7a133b319d01d0fe0e208f385498f009/pytorch3d/ops/knn.py#L133
